vllm/sequence.py

```diff
@@ -142,7 +142,7 @@ def __init__(
         prompt: str,
         prompt_token_ids: List[int],
         block_size: int,
-        eos_token_id: int,
+        eos_token_id: Optional[int] = None,
```

Comment:
cc @njhill FYI https://github.com/vllm-project/vllm/pull/3166

Reply:
Thanks @cadedaniel and apologies for missing these tests in the original PR.